### DISCUSSION TITLE
Operator Precedence

### DISCUSSION THREAD

**Discussion by inguin** [lcn2/calc#74] [Q&A] [answered]
First of all: Thanks a lot for this awesome tool! I've been using it almost daily for at least a decade, probably more.

The help describes the `calc` language as "C-like". I wonder how compatible it aims to be.

Background: In the C language, the `*` operator has a higher precedence than `<<`, so `3 << 2*5` evaluates to 3072. In `calc` the result is 60.

**Comment by lcn2** [comment]
We are on an eclipse expedition in a very remote area of the world with nil to none internet access. We won't be back in internet range until after 2023 May 20. Please pardon the delay ...

This message was sent via a slow satellite link and we are likely to not see any replies until after 2023 May 23.

**Reply by lcn2** [reply, marked as answer]
Thank you @inguin.  You are correct that this is a difference between calc and C.  The master branch includes text added to help/unexpected so that `help unexpected` documents this difference.

This change will be in the next release of calc.
